### ISSUE TITLE
Skill Points Bug

### ISSUE THREAD

**Issue by Fable1991** [BarleyFlour/RespecMod#58]
Used the mod to respec Delamere and just leveled up for the first time since doing it. They leveled up to 13 but for the skill points only it thinks Delamere leveled to 23. She didn't get 10 actual levels, just 54 skill points to spend and the cap on each skill is set to 23.
 
To clarify, she is not the only character that I respecced, but she is the only one to have the problem. Obviously it's not a big deal, it just clearly wasn't intended. Just double checked it and not that I've finished leveling her up, it cut off any points that were above 13. The extra points I put into skills below 13 are still there and working though.
  
Also, I don't know if this next one is on your end or Owlcat's, but also after respeccing Delamere and Staunton they both still seem to count as having the feats that they had before. They aren't in their character pages but they count for feats that need them as pre-requisites. I haven't tested them in combat to see if they all work properly.
 
Just remembered a third one, everyone who I respecced (and that's most of my companions) lost their background and gods. They all just say "none" for both. Not atheism, just no god selected. Especially weird for Staunton since he's a warpriest and his abilities still work.

![Diety example](https://user-images.githubusercontent.com/90477259/132969590-b41dd436-b993-417a-8a40-9fa2017be315.png)

![Background example](https://user-images.githubusercontent.com/90477259/132969595-1adb21ec-2cab-4cc2-9b13-2051407e7bec.png)

**Comment by uniwars** [comment]
I also got the skill problem with nenio
I can add that nenio is also 1 level further then my group when leveling, but only when leveling
I have respe her into arcanist, but each time she gains a level she also get her first scroll wizard level in the background, so she gets extra ability, feat, etc in advance from my group and even when she then reach the correct level the feats I had chosen previously are not added (when I get level 12 I took her a 13 level feat, she didn't get it, even when my group was 13 and her 14 in the level-up window) -after leveling up she has her correct level and class, it's just in the leveling window)

**Comment by BarleyFlour** [comment]
is this with 1.05.3?

**Comment by Fable1991** [comment]
> is this with 1.05.3?

It was 1.05.2

**Comment by Fable1991** [comment]
Update: Just got to level 14 and the extra skill points thing happened again, and again only to Delamere.

**Comment by BarleyFlour** [comment]
Could you send a save file?

**Comment by Fable1991** [comment]
> Could you send a save file?

Absolutely, what's the best way to do that? It wont let me post it here because it's the wrong file type.

**Comment by BarleyFlour** [comment]
You should be able to just zip it up (right click > send to > Compressed (zipped) folder) and send it here.

**Comment by Fable1991** [comment]
[Manual_55_Lower_City__27_Gozran__IV__4716__15_36_45.zip](https://github.com/BarleyFlour/RespecMod/files/7158556/Manual_55_Lower_City__27_Gozran__IV__4716__15_36_45.zip)
